tox.ini

```diff
@@ -1,6 +1,8 @@
 [tox]
 requires = 
-    tox >=4.9.0, <4.14.1
+    tox >=4.9.0
+    # https://github.com/tox-dev/tox/issues/3238
+    tox < 4.14.1
```

Comment:
ok, this is great! I did not realized we could do it, I'll add it to all the other repos